pages/linux/rpmspec.md

```diff
@@ -7,6 +7,10 @@
 
 `rpmspec --query {{path/to/rpm.spec}}`
 
+- List all queryformat options:
```

Comment:
Perhaps this better
```suggestion
- List all `--queryformat` options:
```

Reply:
Or maybe
```suggestion
- List all options for `--queryformat`:
```

Reply:
Thank you for the review. This version looks better :)